release_notes.md

```diff
@@ -3,7 +3,13 @@
 <!-- Please add your release notes in the following format:
 - My change description (#PR)
 -->
+<<<<<<< HEAD
```

Comment:
This is an artifact from the merge commit - please carefully compare the difference between the release notes in the release branch and the release notes updates in Gavin's PR.